libnavigation-base/src/main/java/com/mapbox/navigation/base/options/EHorizonOptions.kt

```diff
@@ -79,31 +79,41 @@ class EHorizonOptions private constructor(
         private var length: Double = 500.0
         private var expansion: Int = 0
         private var branchLength: Double = 50.0
-        private var includeGeometries: Boolean = false
+        private var minTimeDeltaBetweenUpdates: Double? = null
 
         /**
-         * Override he minimum length of the EHorizon ahead of the current position.
+         * Override the minimum length of the EHorizon ahead of the current position.
+         * If negative default value will be used.
          */
         fun length(length: Double): Builder =
-            apply { this.length = length }
+            apply { this.length = if (length >= 0) length else DEFAULT_LENGTH }
```

Comment:
I believe we need to throw an exception because a developer explicitly sets the non-valid value. Here and below.

Reply:
discussed with NN team. we will set a default value. NN team going to implement options validation on their side in the future

Reply:
If that a definitive error, it feels like we should be fine with throwing an exception, especially since it's in an options class. This will be caught very quickly during any type of intergation.

Reply:
done, changed to exceptions, tests updated